application/models/Kalkun_model.php

```diff
@@ -363,9 +363,12 @@ function update_setting($option)
 				break;
 
 			case 'password':
-				$this->db->set('password', password_hash($this->input->post('new_password'), PASSWORD_BCRYPT));
-				$this->db->where('id_user', $this->session->userdata('id_user'));
-				$this->db->update('user');
+				if ( ! ($this->config->item('demo_mode') && intval($this->session->userdata('id_user')) === 1))
```

Comment:
Demo mode = true and for non default user, this will change the password. Should we disable all password change?

Reply:
I thought we might let the users play around by adding users and also the possibility to change the password of the users.

Reply:
Hmm, sounds cool!